x-pack/plugins/cases/server/client/utils.ts

```diff
@@ -503,7 +503,7 @@ export const constructSearch = (
   search: string | undefined,
```

Comment:
In line 485 of this file we do

```
    case 'closedAt':
    case 'closed_at':
      return SortFieldCase.closedAt;
```

I think the `case 'closed_at':` is now unnecessary.

I took a look and there seem to be only a couple of places where this sorting field was explicitly being used:
1 - the security solution usage you fixed.
2 - when we call find in `use_cases_list`, but there we already passed `createdAt`

Reply:
Ahh okay, then I will remove it

Reply:
I saw `closed_at` is used somewhere in telemetry `x-pack/plugins/cases/server/telemetry/queries/cases.ts`.
So I think it's better to keep it, just in case 🤷‍♀️

Reply:
Maybe we can change it there too? the same way the security solution plugin was changed 😛 If we're fixing it once and for all might as well do it everywhere

Reply:
hmm.. makes sense, then should I also update `created_at` and `updated_at` everywhere?

Reply:
Ok, I had a call with @cnasikas and we also discussed this a bit. This PR introduces schema validation for the sort field restricting the possible values with `CasesFindRequestSortFieldsRt`.

This validation only accepts   `createdAt`, `updatedAt`, and `closedAt` which means the verification fails if any API or plugin calls this with the old values `created_at`, `updated_at`, and `closed_at`. So it seems we actually need to change those in this PR 😅

Reply:
okay, thanks for the update. I will make the changes 😅